examples/pidof.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+"""
+A clone of 'pidof' cmdline utility.
+$ pidof  /usr/bin/python
+1140 1138 1136 1134 1133 1129 1127 1125 1121 1120 1119
+"""
+
+import psutil
+import sys
+
+
+def pidsof(pgm):
+    pids = []
+    for proc in psutil.process_iter():
+        process = proc.as_dict()
```

Comment:
there's no need to call as_dict() (which retrieves *all* process info; instead just call proc.cmdline()